README.md

```diff
@@ -107,6 +111,14 @@ rendering your Ember.js application using the [FastBoot App Server](https://gith
 Run `fastboot` with the `DEBUG` environment variable set to `fastboot:*`
 for detailed logging.
 
+### Result
+
+The result from `fastboot` is a `Result` object that has the following API:
+
+1. `html`: this returns the serialized DOM contents after running the route on server side
+2. `domContents` : This returns an object that contains the `head` and `body` of the server side rendered content in a serialized string
+3. `getAnalytics`: This returns any request specfic analytical information that can be leveraged for monitoring the application. Example: when using `buildSandboxPerVisit`, this returns if the request ran in a degraded fashion when the queue was starved.
```

Comment:
Thoughts on tweaking this to be in a TypeScript interface presentation?

```ts
interface FastBootVisitResult {
  /**
    The serialized DOM contents after completing the `visit` request.

    Note: this combines the `domContents.head` and `domContents.body`.
  */
  html: string;
  
  domConents: {
    /**
      The `<head>` contents generated by the visit.
    */
    head: string;

    /**
      The `<body>` contents generated by the visit.
    */
    body: string;
  };
}
```

I do not fully understand the `getAnalytics` part of the result. Is it a function the user has to call? Is it an object?

Reply:
That is a good suggestion. I'll update it. 

Yes `getAnalytics` is a public function that the user calls to get any request specific info that was built during `visit`. The return value is an object. Alternatively, you could just call `result.analytics` as well, this just provides a getter function.

In future, I want to add more timing info on time took for `visit` and other measurements that apps can track.